### ISSUE TITLE
ANE Structure

### ISSUE THREAD

**Issue by mspdevelopers** [rainbowcreatures/FlashyWrappers#16]
The ANE structure is separated by platform, it could be helpful if we can merge the mobile ANE in one that supports:
 Android-ARM
 Android-x86
 iOS-ARM
 iOS-x86

Another ANE for Desktop, supporting:
Windows-x86
Windows-x64
MacOS-x86
MacOS-x64

And if it's possible each ANE has a fallback to the flash-player swc, so we have to set the bridge swf by default and it will use it by context.

So at the end we can have only one application.xml that includes the ANE for mobile and it supports the flash-player fallback too.

**Comment by rainbowcreatures** [comment]
Yeah it would be great to merge into single ANE...see my previous answer though.
